libraries/AP_HAL_ChibiOS/hwdef/luminousbee5/hwdef.dat

```diff
@@ -165,10 +183,13 @@ PE15 MAG_CS CS
 SPIDEV dps310      SPI2 DEVID3  BARO_CS           MODE3  5*MHZ  5*MHZ
 #SPIDEV icm20608    SPI1 DEVID6  20608_CS   MODE3   2*MHZ   8*MHZ
 SPIDEV hmc5843     SPI1 DEVID5  MAG_CS     MODE3  11*MHZ  11*MHZ
-SPIDEV bmi088_g    SPI1 DEVID1  BMI088_GYRO_CS    MODE3 10*MHZ 10*MHZ
-SPIDEV bmi088_a    SPI1 DEVID2  BMI088_ACCEL_CS   MODE3 10*MHZ 10*MHZ
+#SPIDEV bmi088_g    SPI1 DEVID1  BMI088_GYRO_CS    MODE3 10*MHZ 10*MHZ
+#SPIDEV bmi088_a    SPI1 DEVID2  BMI088_ACCEL_CS   MODE3 10*MHZ 10*MHZ
 SPIDEV ramtron     SPI2 DEVID10 FRAM_CS    		MODE3   8*MHZ   8*MHZ
-SPIDEV dwm1000     SPI4 DEVID5  UWB_CS     		MODE0   3*MHZ  20*MHZ
+#SPIDEV dwm1000     SPI4 DEVID5  UWB_CS     		MODE0   3*MHZ  20*MHZ
+SPIDEV icm42605     SPI4 DEVID4  IMU_CS     		MODE3   16*MHZ  16*MHZ
```

Comment:
You usually want the lower speed to be safe. So maybe this should be something like 2*Mhz 16*Mhz

Reply:
I will report this to my colleague , thanks

Reply:
Resolved now